src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6247,3 +6247,32 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_gelu_float_tanh) {
     test_case.add_input<float>(Shape{2}, {-0.5f, 24.33f});
     test_case.add_expected_output<float>(Shape{2}, {-0.15428598f, 24.f});
 }
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_softmax_cross_entropy_loss_int) {
+    auto function = onnx_import::import_onnx_model(file_util::path_join(ov::test::utils::getExecutableDirectory(),
+                                                                        SERIALIZED_ZOO,
+                                                                        "onnx/softmax_cross_entropy_loss_int.onnx"));
+
+    auto test_case = ov::test::TestCase(function, s_device);
+    test_case.add_input<int64_t>(Shape{2}, {1, 2});
+    test_case.add_input<int64_t>(Shape{2}, {0, 2});
+    test_case.add_expected_output<bool>(Shape{2}, {false, true});
```

Comment:
Why the type of the expected output values is `bool`?
According to the ONNX [SoftmaxCrossEntropyLoss](https://onnx.ai/onnx/operators/onnx__SoftmaxCrossEntropyLoss.html#outputs) specification, it should be floating-point precision.